apple-pie/index.html

```diff
@@ -3,37 +3,45 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Apple Pie Recipe</title>
+    <title>Apple Pie Recipe</title> <!---->
+    <link rel= "stylesheet" href="style.css"/>
     <!-- Remember to link your styles -->
   </head>
   <body>
-    Apple Pie
+    <section id="section" class="section">
+    <div>
+    <h1 class= "applepie">Apple Pie</h1>
+    </div>
+    </section>
+    <p>This was my grandmother's apple pie recipe. I have never seen another one <br> quite like it. It will always
+    be my favorite and has won me several first place <br> prizes in local competitions. I hope it becomes one of your
+    favorites as well!</p> 
+   <br>
+    <img src="images\recipe-info.png" alt="recipe-info" height="70px"/> <!--height="100px"-->
+   
+    <ul><h3>Ingredients</h3> 
+      <li>1 recipe pastry for a 9 inch double crust pie</li> 
+      <li>1/2 cup unsalted butter</li> 
+      <li>3 tablespoons all-purpose flour</li>
+      <li>1/4 cup water</li>
+      <li>3 tablespoons all-purpose flour</li>
+      <li>1/4 cup water 1/2 cup white sugar</li>
+      <li>1/2 cup packed brown sugar</li>
+      <li>8 Granny Smith apples - peeled, cored and sliced</li>
+    </ul>  
 
-    This was my grandmother's apple pie recipe. I have never seen another one quite like it. It will always
-    be my favorite and has won me several first place prizes in local competitions. I hope it becomes one of your
-    favorites as well! 
+    <img src="images\cooking-info.png" alt="recipe-info" height="90px"/> <!--height="100px"-->
     
-    Ingredients 
-    
-    1 recipe pastry for a 9 inch double crust pie 
-    1/2 cup unsalted butter 
-    3 tablespoons all-purpose flour
-    1/4 cup water
-    3 tablespoons all-purpose flour
-    1/4 cup water 1/2 cup white sugar
-    1/2 cup packed brown sugar
-    8 Granny Smith apples - peeled, cored and sliced
-
-    Directions
+    <h3>Directions</h3>
 
-    Preheat oven to 425 degrees F (220 degrees C).Melt the butter in a saucepan. Stir in flour to form a paste. Add water, white sugar and brown sugar, and bring
-    to a boil. Reduce temperature and let simmer. 
+    <p>1)Preheat oven to 425 degrees F (220 degrees C).Melt the butter in a saucepan. Stir in <br> flour to form a paste. Add water, white sugar and brown sugar, and bring
+    to a boil. Reduce <br> temperature and let simmer. <br>
 
-    Place the bottom crust in your pan. Fill with apples, mounded
-    slightly. Cover with a lattice work crust. Gently pour the sugar and butter liquid over the crust. Pour slowly so
-    that it does not run off.
+    2)Place the bottom crust in your pan. Fill with apples, mounded
+    slightly. Cover with a <br> lattice work crust. Gently pour the sugar and butter liquid over the crust. Pour slowly so <br>
```

Comment:
Don't use `<br>` in here, lines will break when you add a class for this section and put some margins in it